src/formatter.cc

```diff
@@ -28,15 +28,15 @@
 
 namespace toggl {
 
-const std::string Format::Classic = std::string("classic");
-const std::string Format::Improved = std::string("improved");
-const std::string Format::Decimal = std::string("decimal");
+const std::string &Format::Classic = std::string("classic");
+const std::string &Format::Improved = std::string("improved");
+const std::string &Format::Decimal = std::string("decimal");
```

Comment:
does it make sense to use references here?

Reply:
Nice catch! While not breaking anything, it doesn't really make sense, you're right. I'll get rid of it.